### DISCUSSION TITLE
About texture switching of 3D objects

### DISCUSSION THREAD

**Discussion by zgw426** [pyvista/pyvista#1528] [Q&A] [answered]
Is it possible to change the texture of a 3D object texture drawn in PyVista over time?
I want to prepare multiple (100 or more) images and switch the image to be mapped as a texture every second, but I'm having a problem because I don't know how to do it. I would be grateful if you could let me know if you have any idea.

**Comment by tkoyama010** [comment, marked as answer]
@zgw426  Sorry for our late reply. I added the example in https://github.com/pyvista/pyvista/pull/1544 . We can see how to update the texture of 3D object.
